lib/logger_json.ex

```diff
@@ -172,6 +172,14 @@ defmodule LoggerJSON do
   defp put_env(env), do: Application.put_env(:logger_json, :backend, env)
 
   defp meet_level?(_lvl, nil), do: true
+  defp meet_level?(lvl, lvl), do: true
```

Comment:
This clause will always match and the ones we conditionally add below won't be ever used

Reply:
Hi @AndrewDryga thanks for your feedback.

> But I have a strong feeling that I want to rewrite this library as a logger formatter now :)

I would like that very much! If you want any help, I am available to do some contributions.

> This clause will always match and the ones we conditionally add below won't be ever used

This is when both values ares equal:

```elixir
meet_level?(lvl, lvl)
# is equal to
meet_level?(lvl, lvl2) when lvl == lvl2
```

If they are not equal (and if the second parameter is not `nil`), it will continue through the other conditionals.

Reply:
Ahh, yes, sorry, I did not notice that